File: myproj26/main03.py


# 사전
tom = {
    "name": "Tom",
    "age": 10,
    # = "ag" + "e": 10,
    # age: 10,
}

name = "Tom"
age = 10

tom1 = {
    "name": name,
    "age": age,

}

# f-string = String Interpolation이라고 한다.
print(f"안녕나는 {name}이야/ {age}살이지.")

# 두 줄 쓰는 """
print(f"""안녕나는 {name}이야
 {age}살이지.""")

File: myproj26/main03.js


// 객체
const age1 = "나이";
const tom = {
    "name": "Tom",
    // "age": 10,
    // age: 10,
    [age1]: 10,

}

console.log(tom);


const name = "Tom";
const age = 10;
const tom1 = {
    name,
    age,
    print: function () {
        // console.log(this.name, this.age);
        // Template Literals라고한다. 줄바꿈은 그냥 엔터
        console.log(`안녕, 나는 ${this.name}이야. ${this.age}살이지.`);
    }
}

tom1.print();